libkbfs/data_types.go

```diff
@@ -1099,3 +1099,10 @@ func (bra BlockRequestAction) CacheType() DiskBlockCacheType {
 	}
 	return DiskBlockAnyCache
 }
+
+func defaultBlockRequestAction(mode InitMode) BlockRequestAction {
```

Comment:
why isn't this a function on `Mode()`?

Reply:
Because I didn't think about it.  Fixed.